landscaper/gardenlet/pkg/gardenlet.go

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	v2 "github.com/gardener/component-spec/bindings-go/apis/v2"
+	"github.com/gardener/component-spec/bindings-go/apis/v2/cdutils"
+	"github.com/gardener/component-spec/bindings-go/codec"
+	landscaperv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	landscaperconstants "github.com/gardener/landscaper/apis/deployer/container"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	confighelper "github.com/gardener/gardener/pkg/gardenlet/apis/config/helper"
+	configv1alpha1 "github.com/gardener/gardener/pkg/gardenlet/apis/config/v1alpha1"
+	"github.com/gardener/gardener/pkg/logger"
+
+	"github.com/gardener/gardener/landscaper/gardenlet/pkg/apis/imports"
+)
+
+const (
+	gardenerComponentName = "github.com/gardener/gardener"
+	gardenletImageName    = "gardenlet"
+)
+
+// Landscaper has all the context and parameters needed to run a Gardenlet landscaper.
+type Landscaper struct {
+	log                      *logrus.Entry
+	gardenClient             kubernetes.Interface
+	seedClient               kubernetes.Interface
+	// using internal version of the import configuration
+	Imports                  *imports.Imports
```

Comment:
```suggestion
	imports                  *imports.Imports
```
Does this field really need to be exported?